src/utils/testData.ts

```diff
@@ -0,0 +1,206 @@
+export function getRandomName() {
+	return namesArray[Math.floor(Math.random() * namesArray.length)]
+}
+
+export const namesArray = [
```

Comment:
q/todo: Why do you export this? It's not used in any other file.

nit: The file could be `utils.ts`, start simple, no need to give specific names. And also, this ain't a file related with tests :)

Reply:
It is used in the CardsContainer, which I have not committed the changes yet. I guess it should be called sampleData.ts as it is a sample of names to use on the user cards.

Reply:
I changed the name. And the last commit has all the files, including the one that uses the namesArray. DONE.